.github/ISSUE_TEMPLATE/Feature_request.yml

```diff
@@ -0,0 +1,22 @@
+name: Feature Request
+description: Suggest an idea for SciPy
+title: "[Feature]: "
+labels: [enhancement]
+
+body:
+  - type: input
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+    validations:
+      required: true
```

Comment:
I would not make this required as the feature can also not be issue related.

Reply:
Fixed!